frontend/src/components/common/NavBar.tsx

```diff
@@ -16,10 +17,12 @@ import authAPIClient from "../../APIClients/AuthAPIClient";
 import logo from "../../assets/ccbc.png";
 import { UserRole } from "../../constants/Enums";
 import AuthContext from "../../contexts/AuthContext";
+import ChangePasswordModal from "./ChangePassword/ChangePasswordModal";
 
 const NavBar = (): React.ReactElement => {
   const { authenticatedUser, setAuthenticatedUser } = useContext(AuthContext);
   const isAdmin = authenticatedUser?.roleType === UserRole.Admin;
+  const { isOpen, onOpen, onClose } = useDisclosure();
```

Comment:
nit: rename these variables to be more specific to the change password modal, e.g. `isChangePasswordModalOpen`, etc.

Reply:
Not really necessary if we are certain that there won't be other modals on the `NavBar`, but just incase I think it would be helpful.